.github/workflows/build-master.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Master
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    # <minute [0,59]> <hour [0,23]> <day of the month [1,31]> <month of the year [1,12]> <day of the week [0,6]>
+    # https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07
+    # Run every Monday at 18:00:00 UTC (Monday at 10:00:00 PST)
+    - cron: '0 18 * * 1'
```

Comment:
just curious, why run weekly?

Reply:
I think Jackson set this up similar to aicsimageio so might be an artifact of that?